bugbug/rust_code_analysis_server.py

```diff
@@ -45,7 +45,7 @@ def start_process(self):
 
         try:
             self.proc = subprocess.Popen(
-                ["rust-code-analysis", "--serve", "--port", str(self.port),]
+                ["rust-code-analysis", "--serve", "--port", str(self.port)]
```

Comment:
WTH, why is black giving different results for you and me? :P

Reply:
I don't know... my editor was open before black ran and the comma is there... so it removed it